backend/lib/rpc-service/data-types.js

```diff
@@ -0,0 +1,41 @@
+import { Type } from '@sinclair/typebox'
+const ClaimEvent = Type.Object({
+  id: Type.Number(),
+  client: Type.Number(),
+  provider: Type.Number(),
+  pieceCid: Type.Any(),
+  pieceSize: Type.Number(),
```

Comment:
Please use the BigInt type here to match the type found in the RPC API responses.